api-test/src/test/resources/packages.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+	Defines the EMR metadata packages
+-->
+<org.openmrs.module.metadatasharing.metadata.MetadataPackagesConfig>

```

Comment:
Actually, I think we should remove this file (with the full package name) and have the "standard" test the one one with the short aliased name.
Basically, the official, supported behavior going forward is just `<MetadataPackagesConfig>`, so that's the only thing we test.
(The full name will still work, but let's not advertise it.)